releasenotes/notes/update-cs-csdg-in-equivalence-library-c1e70d3246f4aa6d.yaml

```diff
@@ -0,0 +1,7 @@
+---
+upgrade_synthesis:
```

Comment:
If we add additional equivalences (see comment above) then I think this is now actually a new feature, instead of an upgrade, or what do you think?